applications/nrf_desktop/src/hw_interface/motion_sensor.c

```diff
@@ -463,8 +484,13 @@ static bool event_handler(const struct event_header *eh)
 		switch (state.state) {
 		case STATE_FETCHING:
 		case STATE_IDLE:
+		case STATE_DISCONNECTED:
 			enable_trigger();
-			state.state = STATE_SUSPENDED;
+			if (state.state == STATE_DISCONNECTED) {
```

Comment:
Maybe we could move the code around and remove if ?

Reply:
I was thinking about it but decided not too. This is to not duplicate the code.

Reply:
thought about it but decided not to to avoid duplication